checkClock/db/migrate/20220113230211_create_admins.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAdmins < ActiveRecord::Migration[6.1]
+  def change
+    create_table :admins do |t|
+      t.string :email
+      t.string :password
```

Comment:
Consider using `Devise` for your authentication system
It's a battle-tested gem that provides the auth mechanism to your rails app, instead of figuring yourself how to make it secure 😄 👍🏽

Reply:
We'll definitely look into it, and try to incorporate it into our next sprint